### PR TITLE
OSDOCS-9793: Doc edits in storage vol group and getting started

### PR DESCRIPTION
**Pull request by aravipra**
Version(s):
4.14, 4.15 and 4.16

Issue:
[OSDOCS-9793](https://issues.redhat.com/browse/OSDOCS-9793)

Link to docs preview:
[Architecture](https://72443--ocpdocs-pr.netlify.app/microshift/latest/microshift_getting_started/microshift-architecture)
[volume group name](https://72443--ocpdocs-pr.netlify.app/microshift/latest/microshift_storage/microshift-storage-plugin-overview#lvms-volume-group-name_microshift-storage-plugin-overview)

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

File: microshift_getting_started/microshift-architecture.adoc [hunk 1]
```diff
@@ -43,7 +43,7 @@ The following operational differences from {oke} can help you understand where {
 .{product-title} differences from {oke}.
 image::311_RHDevice_Edge_Overview_0223_2.png[<{microshift-short} is tasked with only the Kubernetes cluster capabilities of networking, ingress, storage, helm, with the additional Kubernetes functions of orchestration and security, as the following diagram illustrates.>]
 
-Figure 2 shows that {oke} has the same cluster capabilities as {product-title}, and adds the following:
+Figure 2.2 shows that {oke} has the same cluster capabilities as {product-title}, and adds the following:
```
Comment (stevsmit) [inline microshift_getting_started/microshift-architecture.adoc:46]: Should the actual image be titled 2.2? On the same page, I see you're referencing "Figure 2.2," however, I do not see a "Figure 2.2". I see a "Figure 2". 

We need a noun after "following". For example: 

```suggestion
Figure 2.2 shows that {oke} has the same cluster capabilities as {product-title}, and adds the following information:
```

File: modules/microshift-lvms-system-requirements.adoc [hunk 2]
```diff
@@ -11,9 +11,10 @@ Using LVMS in {microshift-short} requires the following system specifications.
 [id="lvms-volume-group-name_{context}"]
 == Volume group name
 
-If you did not configure LVMS in an `lvmd.yaml` YAML file placed in the `/etc/microshift/` directory, {microshift-short} attempts to assign a default volume group (VG) dynamically by running the `vgs` command. 
-* {microshift-short} assigns a default VG when only one VG is found. 
-* If more than one VG is present, the VG named `microshift` is assigned as the default. 
+If you did not configure LVMS in an `lvmd.yaml` YAML file placed in the `/etc/microshift/` directory, {microshift-short} attempts to assign a default volume group (VG) dynamically by running the `vgs` command.
```
Comment (stevsmit) [inline modules/microshift-lvms-system-requirements.adoc:14]: Might want to consider defining LVMS in the first usage.

```suggestion
If you did not configure LVMS in an `lvmd.yaml` file placed in the `/etc/microshift/` directory, {microshift-short} attempts to assign a default volume group (VG) dynamically by running the `vgs` command.
```

**Comment by stevsmit** [comment]
I see that @rh-tokeefe is also reviewing so I'll let him add the peer-review-done label.

**Comment by rh-tokeefe** [comment]
@stevsmit I'm still onboarding through the OCP review process. I thought I placed the peer review in progress label on this while I was working. Sorry if I missed that you were already looking at it. 

With your comments, this LGTM.

File: microshift_getting_started/microshift-architecture.adoc (hunk 1 above)
Comment (aravipra) [reply microshift_getting_started/microshift-architecture.adoc:46]: Actually the preview shows Figure 2 but the actual customer portal shows Figure 2.2. So I have removed the numbering and added the title to resolve this issue.

File: modules/microshift-lvms-system-requirements.adoc (hunk 2 above)
Comment (aravipra) [reply modules/microshift-lvms-system-requirements.adoc:14]: This is not part of the changes made in this jira. 
Also should "YAML" be removed in line 14?

File: microshift_getting_started/microshift-architecture.adoc [hunk 3]
```diff
@@ -43,7 +43,7 @@ The following operational differences from {oke} can help you understand where {
 .{product-title} differences from {oke}.
 image::311_RHDevice_Edge_Overview_0223_2.png[<{microshift-short} is tasked with only the Kubernetes cluster capabilities of networking, ingress, storage, helm, with the additional Kubernetes functions of orchestration and security, as the following diagram illustrates.>]
 
-Figure 2 shows that {oke} has the same cluster capabilities as {product-title}, and adds the following:
+Figure {product-title} differences from {oke} shows that {oke} has the same cluster capabilities as {product-title}, and adds the following:
```
Comment (ShaunaDiaz) [inline microshift_getting_started/microshift-architecture.adoc:46]:
```suggestion
The figure "{product-title} differences from {oke}" shows that {oke} has the same cluster capabilities as {product-title}, and adds the following information:
```

Comment (aravipra) [reply microshift_getting_started/microshift-architecture.adoc]: Done

**Pull request merged**